Small Medbay Remap

Ryan180602 opened the pull request:
:cl: Ryan180602
maptweak: Change Physician's Paperwork Office to Medical Paperwork Office (for all medical personnel)
maptweak: Remapped Exam Room
maptweak: Added 3 Runtime spawn locations
/:cl:

To facilitate usage of the paperwork office more (for antag and non-antag purposes).

To change the console around and make it look a *bit* better.

*More Runtime*.

SierraKomodo commented on maps/torch/torch5_deck1.dmm:
```diff
@@ -6549,7 +6549,7 @@
 "aqT" = (
 /obj/machinery/door/airlock/medical{
-	name = "Physicians Paperwork Office"
+	name = "General Paperwork Office"
```
To avoid 'Where is this?' confusion with area names showing up in places like alerts.

```suggestion
	name = "Medical Paperwork Office"
```

Ryan180602 committed:
Small MedBay Remap

Added bonus Runtime multipoints, changed physician's paperwork office to medical paperwork office (all med-access), remapped exam room slightly

Spookerton reviewed (approved):
Code's fine, but I'm not really a fan of the exam room change. Cramps the working bits and adds a column of empty space that seems pointless.
